src/platform/Linux/NetworkCommissioningDriver.cpp

```diff
@@ -0,0 +1,269 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <lib/support/CodeUtils.h>
+#include <lib/support/SafeInt.h>
+#include <platform/CHIPDeviceLayer.h>
+#include <platform/Linux/NetworkCommissioningDriver.h>
+#include <platform/Linux/ThreadStackManagerImpl.h>
+#include <platform/ThreadStackManager.h>
+
+#include <string>
+#include <vector>
+
+using namespace chip;
+using namespace chip::Thread;
+
+namespace chip {
+namespace DeviceLayer {
+namespace NetworkCommissioning {
+
+#if CHIP_DEVICE_CONFIG_ENABLE_WPA
+CHIP_ERROR LinuxWiFiDriver::CommitConfiguration()
+{
+    mRunningNetwork = mStagingNetwork;
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR LinuxWiFiDriver::RevertConfiguration()
+{
+    mStagingNetwork = mRunningNetwork;
+    return CHIP_NO_ERROR;
+}
```

Comment:
This should show the calls needed to do persistent storage. Does `ConnectivityMgrImpl().ProvisionWiFiNetwork` do storage? If so, then it's the opposite: RevertConfiguration will not work.

Reply:
ProvisioningWiFiNetwork is Connect actually, the actual persistent operation is highly platform dependent.

Reply:
Made some change to ConnectivityManagerImpl